cloud/scope/cluster.go

```diff
@@ -400,6 +388,16 @@ func (s *ClusterScope) Location() string {
 	return s.AzureCluster.Spec.Location
 }
 
+// FailureDomains returns the failure domains.
+func (s *ClusterScope) FailureDomains() clusterv1.FailureDomains {
+	return s.AzureCluster.Status.FailureDomains
+}
+
+// DeletionTimeStamp returns the deletion timestamp.
```

Comment:
```suggestion
// DeletionTimeStamp returns the deletion timestamp of the owner Cluster.
```

for clarity

Reply:
This code should have been removed. Fixed it, Thanks.